### DISCUSSION TITLE
GitHUB Connect Tutorial

### DISCUSSION THREAD

**Discussion by tidypy** [DaxStudio/DaxStudio#833] [Ideas]
PBI has a new feature to Connect or "Get Data" Via GitHUB.  
The DAXstudio Tutorial can use this as a repo or simulate Azure in this way.

**Comment by dgosbell** [comment]
I don't think the Github connector really helps much in terms of the tutorial. DAX Studio connects to the data model after the data has been loaded. It does not really care where the data gets loaded from.
